modules/installation-mirror-repository.adoc

```diff
@@ -75,17 +153,48 @@ imageContentSources:
 - mirrors:
   - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
   source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+endif::install[]
+ifdef::update[]
+To use the new mirrored repository for upgrades, use the following to create an ImageContentSourcePolicy:
+
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: example
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-release
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+endif::update[]
 ...
 ----
 
-. Record the `imageContentSources` section from the output of the previous
-command. This information is required during {product-title} installation.
+. Record the
+ifdef::install[]
+`imageContentSources`
+endif::install[]
+ifdef::update[]
+`ImageContentSourcePolicy`
+endif::update[]
+section from the output of the previous
+command. This information is required
+ifdef::install[]
+during {product-title} installation.
+endif::install[]
+ifdef::update[]
+when you update your {product-title} cluster.
+endif::update[]
 
+ifdef::install[]
 . To create the installation program that is based on the content that you
 mirrored, extract it and pin it to the release:
 +
 ----
-$ oc adm release extract --command=openshift-install "${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}"
+$ oc adm release extract --command=openshift-install "${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${DIGEST}"
```

Comment:
If we decide to use release_version, here should revert 'DIGEST' back.

Reply:
We want `@${DIGEST}` here to guard against mirror-registry mutation.